nats.go

```diff
@@ -145,6 +145,7 @@ var (
 	ErrNotJSMessage                 = errors.New("nats: not a jetstream message")
 	ErrInvalidStreamName            = errors.New("nats: invalid stream name")
 	ErrInvalidDurableName           = errors.New("nats: invalid durable name")
+	ErrInvalidConsumerName          = errors.New("nats: invalid consumer name")
```

Comment:
Add stream too?

Reply:
We already have such error, since we check on AddStream, etc..